src/editor/scroller_widget.cpp

```diff
@@ -24,7 +24,7 @@
 #include "video/video_system.hpp"
 #include "video/viewport.hpp"
 
-namespace {
+namespace CONSTS {
 
 const float TOPLEFT = 16;
 const float MIDDLE = 48;
```

Comment:
Why is it necessary to name the namespace?
And could the `const`s be changed to `constexpr`s?

Reply:
Ah, I should've documented that.

MSVC gave [build errors](https://productionresultssa0.blob.core.windows.net/actions-results/2a8be01b-b1a0-431c-988a-83f65d3719bf/workflow-job-run-58628982-a7ad-5965-3a32-4c1b2d88b7fa/logs/job/job-logs.txt?rsct=text%2Fplain&se=2024-06-27T20%3A50%3A18Z&sig=Wnfv1buWUDTPyc4LXnuIHuTjadufTadLjsILbXGxeu4%3D&ske=2024-06-28T06%3A51%3A57Z&skoid=ca7593d4-ee42-46cd-af88-8b886a2f84eb&sks=b&skt=2024-06-27T18%3A51%3A57Z&sktid=398a6654-997b-47e9-b12b-9515b896b4de&skv=2023-11-03&sp=r&spr=https&sr=b&st=2024-06-27T20%3A40%3A13Z&sv=2023-11-03) on [390137b](https://github.com/SuperTux/supertux/pull/2978/commits/390137b739a37edcdf101357a9a77a7b859fd960) about `SIZE` being an ambiguous symbol, probably because a similarly-named variable is defined in some header. Adding the namespace level prevents the symbol clash and makes MSVC happy.

As for `constexpr`, it's most likely possible, but I haven't checked in detail.

Reply:
const values still have internal linkage even if they are not in an anonymous namespace, so constexpr and const have the same effect here I think.
https://en.cppreference.com/w/cpp/language/cv#Notes
https://stackoverflow.com/a/33402258